### PR TITLE
fix: refactor the setting of default scrape-interval

### PR DESCRIPTION
**Pull request by ansrivas**
Needed to match the DBError rather than the string-comparison, hence the refactor.
Another change is to move the function and invoke it as get_or_set i.e. set to default if it doesnt already exist

### CODE REVIEW

File: src/service/db/organization.rs [hunk 1]
```diff
@@ -23,10 +23,7 @@ use std::sync::Arc;
 // DBKey to set settings for an org
 pub const ORG_SETTINGS_KEY_PREFIX: &str = "/organization/setting";
 
-pub async fn set_org_setting(
-    org_name: &str,
-    setting: &OrganizationSetting,
-) -> Result<(), anyhow::Error> {
+pub async fn set_org_setting(org_name: &str, setting: &OrganizationSetting) -> Result<(), Error> {
```
Comment (hengfeiyang) [inline src/service/db/organization.rs:26]: Here, you can directly use `errors::Result<()>`

File: src/handler/http/request/organization/settings.rs [hunk 2]
```diff
@@ -12,17 +12,17 @@
...
+use crate::common::meta::organization::OrganizationSettingResponse;
+use crate::common::utils::json;
+use crate::service::db::organization::get_org_setting;
+use crate::{
+    common::meta::organization::OrganizationSetting, service::db::organization::set_org_setting,
 };
-use crate::service::db::organization::{get_org_setting, set_org_setting, ORG_SETTINGS_KEY_PREFIX};
+
+use actix_web::{get, post, web, HttpResponse};
+use std::io::Error as StdErr;
```
Comment (hengfeiyang) [inline src/handler/http/request/organization/settings.rs:25]: We can format this section.

File: src/job/mmdb_downloader.rs [hunk 3]
```diff
@@ -75,39 +104,12 @@ pub async fn run() -> Result<(), anyhow::Error> {
...
-                    let mut client = MAXMIND_DB_CLIENT.write().await;
-                    *client = maxminddb_client.ok();
-                    log::info!("Updated geo-json data")
-                }
-                Err(e) => log::error!("failed to download the files {}", e),
-            }
-        } else {
-            log::info!("No change in geo-json data")
-        }
+        run_download_files().await;
```
Comment (hengfeiyang) [inline src/job/mmdb_downloader.rs:112]: You can delete the line `108`

Comment (ansrivas) [reply src/job/mmdb_downloader.rs:110]: fixed

File: src/handler/http/request/organization/settings.rs (hunk 2 above)
Comment (ansrivas) [reply src/handler/http/request/organization/settings.rs]: done

**Pull request merged**